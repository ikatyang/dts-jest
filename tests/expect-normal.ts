declare const expect_normal: () => {result: any};

// @dts-jest 1: expect
expect_normal();

// @dts-jest 2: expect
expect_normal();

// @dts-jest 3: expect
expect_normal();
