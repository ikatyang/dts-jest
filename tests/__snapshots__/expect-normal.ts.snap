// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`1: expect 1`] = `"{ result: any; }"`;

exports[`2: expect 1`] = `"{ result: any; }"`;

exports[`3: expect 1`] = `"{ result: any; }"`;
